Assets/Scripts/BlockBehaviour.cs

```diff
@@ -107,4 +108,22 @@ private void TranslateBlock()
             _isTranslating = false;
         }
     }
+
+    private void OnCollisionEnter(Collision other)
+    {
+        if (other.gameObject.name == "Character")
+        {
+            _isPlayerTouched = true;
+        }
+    }
+
+    public bool GetIsTranslating()
+    {
+        return _isTranslating;
+    }

```

Comment:
Let's rename this `IsTranslating`, `GetIsTranslating` implies that the object/script calling the method is aware of the the internal implementation of a private field called `isTranslating`

Reply:
Maybe it would be better to be `IsTranslating`

Reply:
Changed to IsTranslating.